src/crypto/CHIPCryptoPAL.h

```diff
@@ -39,26 +39,28 @@ namespace Crypto {
 
 constexpr size_t kMax_x509_Certificate_Length = 600;
 
-// TODO: Consider renaming these values to be closer to definisions in the spec:
-// CHIP_CRYPTO_GROUP_SIZE_BYTES
-// CHIP_CRYPTO_PUBLIC_KEY_SIZE_BYTES
 constexpr size_t kP256_FE_Length                  = 32;
 constexpr size_t kP256_ECDSA_Signature_Length_Raw = (2 * kP256_FE_Length);
 constexpr size_t kP256_Point_Length               = (2 * kP256_FE_Length + 1);
 constexpr size_t kSHA256_Hash_Length              = 32;
 
+constexpr size_t CHIP_CRYPTO_GROUP_SIZE_BYTES      = kP256_FE_Length;
```

Comment:
I know it's to align the constant naming with the spec, but having two constants with different naming conventions for a single thing looks very confusing. Can we stick to the `kConstant` naming convention and just add annotate the constants with spec names in comments? E.g.
`constexpr size_t kSHA256_Hash_Length              = 32; // CHIP_CRYPTO_HASH_LEN_BYTES in the spec`

Reply:
A lot of the reason code is hard to review for spec compliance is that it uses different names for things like crypto primitives and constants (which are very well defined in spec!) in the implementation, compared to the spec. I can remove this one if it matters a lot, but `kP256_FE_Length` is actually less telling than CHIP_CRYPTO_GROUP_SIZE_BYTES, and similarly, CHIP_CRYPTO_HASH_LEN_BYTES maps to SHA256 today, but may not in the future.

Reply:
@tcarmelveilleux Worth a followup to get rid of kP256_FE_Length?